src/style/chronograf.scss

```diff
@@ -6,143 +6,146 @@
 // Modules
 @import 'modules';
 
-// Clockface UI Kit
-@import 'src/clockface/styles';
-
 // Components
```

Comment:
This section is alphabetized.

Reply:
as long as there aren't any implicit dependencies on things being loaded in the proper order (i.e. if one stylesheet depends on another stylesheet being imported before it can work) this will work

Reply:
i'll take a quick visual pass through the pages.

Honestly, if any are relying on the order, it should be considered a bug. And i'm hoping to have them surface so we can fix them. But as i said, i will take a quick visual pass to see if anything is glaringly out of place.